CHANGELOG.md

```diff
@@ -10,6 +10,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 **Please note**:
 - Your workflow tasks should use `cumulus-message-adapter-js` version 1.0.10+ to utilize new granule, parentArn, asyncOperationId, and stackName fields on the logs.
 
+### BREAKING CHANGES
+
+- **CUMULUS-1321**
+  - The `region` argument to the `cumulus` Terraform module has been removed
```

Comment:
This was only used in the `archive` module, and is not longer used in that module.